datasets/beheerkaart_cbs_grid/beheerkaart_cbs_grid.json

```diff
@@ -0,0 +1,119 @@
+{
+  "type": "dataset",
+  "id": "beheerkaart_cbs_grid",
+  "title": "beheerkaart_cbs_grid",
+  "status": "beschikbaar",
+  "description": "Integratie van elementen beheerkaart publieke ruimte en landelijk CBSgrid",
+  "version": "0.0.1",
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "100",
```

Comment:
Bijzonder id voor een table. Even goed in de gaten houden of dit overal goedgaat ivm. camel- en snakecasing. De naam is ook niet erg "descriptive". Maar, wellicht is men afhankelijk van deze naam?

Reply:
Ja dit zo'n geval waarbij de naam van de tabel voor issues kan zorgen. Bij nader inzien denk ik dat de DSO-API zal zoeken naar 1_0_0 als tabelnaam. Wat natuurlijk incorrect is. Ik kijk even wat er gebeurd en anders neem ik contact op met de klant